src/transformers/modeling_bart.py

```diff
@@ -648,10 +648,9 @@ def forward(
         # get here for encoder decoder cause of static_kv
         if layer_state is not None:  # reuse k,v and encoder_padding_mask
             saved_state = layer_state.get(self.cache_key, {})
-            if "prev_key" in saved_state:
+            if "prev_key" in saved_state and static_kv:
```

Comment:
what does `static_kv` mean excatly ?

Reply:
Like "no need" to recalculate key-value projections?

Reply:
exactly. Notation copied from fairseq.